test/runtime/logger.go

```diff
@@ -0,0 +1,23 @@
+package runtime
+
+import (
+	"fmt"
+	"time"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+// NewLogger creates a pod that will print `count` lines to stdout, waiting for
+// `delay` between each line.  Lines are of the form "<timestamp> [n] `message`"
+// where n is the number of lines output so far. Once done printing the pod will
+// be idle but will not exit until deleted.
+func NewLogger(namespace, name string, count int, delay time.Duration, message string) *corev1.Pod {
```

Comment:
Can we call this LogGenerator or the like and rename the file.  I think "Logger" i think of logging my error messages or the link

Reply:
Done.